android/src/main/kotlin/com/web3auth/flutter/web3auth_flutter/Web3AuthFlutterPlugin.kt

```diff
@@ -133,6 +131,24 @@ class Web3AuthFlutterPlugin : FlutterPlugin, ActivityAware, MethodCallHandler,
                 }
                 return null
             }
+
+            "getPrivKey" -> {
+                val privKey = web3auth.getPrivkey()
```

Comment:
do we handle exceptions ?

Reply:
Internally native sdk's have exception handling mechanism.